examples/README.md

```diff
@@ -3485,4 +3486,224 @@ functions:
 
 </td>
 </tr>
-</table>
\ No newline at end of file
+</table>
+
+### Order iPhone
+
+#### Description
+
+In this example, we want to implement an iPhone ordering workflow. We receive an order by consuming
+an "Order Request" event. Then we start processing the order. 
+
+The purpose of this example is to show how you can implement parallel execution of payment
+processing and preparing the phone shipment. We take advantage of the [Parallel state](../specification.md#Parallel-State) and its `completionType` property which is set to `and` by default. This means that all parallel state
+branches must complete before the state can transition to the next workflow states.
+
+In addition, we show how we can use compensation for each Parallel state branch. This allows us to compensate
+each branch separately. Again, both shipment preparation
+and payment processing have to complete for the order to be completed successfully. 
+
+For the sake of the example we assume that the "Order Request" event has the following sample payload:
+
+```json
+{
+    "order": {
+        "number": "abcde",
+        "phone": "iPhone",
+        "version": "11",
+        "model": "pro",
+        "customerid" : "12345"
+    }
+}
+```
+
+We are not showing the implementation of the function and events definitions in-line. They are assumed 
+to be present in separate JSON files as shown in the workflow definition.
+
+#### Workflow Diagram
+
+<p align="center">
+<img src="../media/examples/iphone-ordering.png" height="400px" alt="iPhone Ordering Example"/>
+</p>
+
+#### Workflow Definition
+
+<table>
+<tr>
+    <th>JSON</th>
+    <th>YAML</th>
+</tr>
+<tr>
+<td valign="top">
+
+```json
+{
+    "id": "iphoneorder",
+    "name": "iPhone ordering workflow",
+    "version": "1.0",
+    "states": [
+        {
+            "name": "Receive Order Request",
+            "type": "event",
+            "onEvents": [
+                {
+                    "eventRefs": ["Order Request Event"],
+                    "actions": [
+                        {
+                            "functionRef": "Send order receved confirmation"
+                        }
+                    ]
+                }
+            ],
+            "transition": "Process Order"
+        },
+        {
+            "name": "Process Order",
+            "type": "parallel",
+            "branches": [
+                {
+                    "name": "Prepare Shipment",
+                    "subflow": "Check Shipment Status"
```

Comment:
This is confusing, I thought this should be the name of another workflow but this example has a state that seems to match this.  Is this supposed to refer to a workflow or a subflow state?

Reply:
https://github.com/serverlessworkflow/specification/pull/271

Reply:
Got it, I found that later and meant to come back and amend my comment but forgot.